.docksal/addons/phpunit/phpunit

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Any particular reason why this is an addon instead of a regular command?

Reply:
I used the `fin addon install phpunit` as a starter and it put it there--I forgot to move it :)